frontend/src/components/mine/ContactInfo/MineContactInfo.js

```diff
@@ -90,6 +90,12 @@ const defaultProps = {
     });
   }
 
+  toggleMineManagerHistory = () => {
```

Comment:
this logic would be more appropriate in the ViewMineManager component, the parent (MineContactInfo) doesn't need to know if the history is viable or not and once the permittee history logic is added MineContactInfo will get a little cluttered, I would suggest moving:

mineManagerHistoryVisible
tiggleMineManagerHistory()
into ViewMineManager.js

and at this point, mineManagerHistoryVisible can be renamed to isHistoryVisible or something shorter, since it is located in the appropiate file you will no longer need to specify if it is the mineManager or the permittee.